cmd/sync-tags/main.go

```diff
@@ -323,6 +306,31 @@ func main() {
 			glog.Fatalf("Failed to create tag %q: %v", bName, err)
 		}
 		createdTags = append(createdTags, bName)
+
+		// if publishSemverTags is true, create annotated semver tag
+		if *publishSemverTags {
+			semverTag := name[:1] + "0" + name[2:] // replace v1.x.y with v0.x.y
```

Comment:
this mapping of upstream v1.x.y upstream -> v0.x.y staging is kube-specific... most of the other options in this script try to remain generic. @sttts might have thoughts on this

Reply:
> this mapping of upstream v1.x.y upstream -> v0.x.y staging is kube-specific... most of the other options in this script try to remain generic. 

For the record: we discussed this on slack and decided to go with renaming the flag to `--publish-v0-semver` to represent the v1 -> v0 semver mapping.